### PR TITLE
New Features for RN 2.2

### PR DESCRIPTION
**Pull request by bgaydosrh**
This PR adds New Feature bullets edited by PM and Dev.

@ousleyp Steve G. has said to remove the previous (and only) feature bullet as the dashboard is not new in this release. If you had another reason to include it, please flag in peer review.

Can you merge and pick to 4.3 and 4.4 as soon as possible? 

Test Build: http://file.bos.redhat.com/bgaydos/021020_2/cnv/cnv_release_notes/cnv-release-notes.html

Thanks

Bob

### CODE REVIEW

File: cnv/cnv_release_notes/cnv-release-notes.adoc [hunk 1]
```diff
@@ -15,10 +15,20 @@ include::modules/technology-preview.adoc[leveloffset=+2]
 
 == New and changed features
 
-* View high-level information about your virtual machines by
-using the *Virtual Machines* dashboard in the {product-title} web console.
-Access it by navigating to the *Workloads* -> *Virtual Machines* page and
-selecting a virtual machine.
+* Enhancements to workflow for importing a single VMware virtual machine.
+
+* Edit Virtual machine templates. Previously, only virtual machines could be edited.
```
Comment (ousleyp) [inline cnv/cnv_release_notes/cnv-release-notes.adoc]: s/Virtual/virtual

File: cnv/cnv_release_notes/cnv-release-notes.adoc [hunk 2]
```diff
@@ -15,10 +15,20 @@ include::modules/technology-preview.adoc[leveloffset=+2]
 
 == New and changed features
 
-* View high-level information about your virtual machines by
-using the *Virtual Machines* dashboard in the {product-title} web console.
-Access it by navigating to the *Workloads* -> *Virtual Machines* page and
-selecting a virtual machine.
+* Enhancements to workflow for importing a single VMware virtual machine.
```
Comment (ousleyp) [inline cnv/cnv_release_notes/cnv-release-notes.adoc]: Please make these bullet points all parallel in structure and make sure to use articles where appropriate ("the workflow"). I like full sentences better than fragments. I also think you should group the similar bullet points together, like:

== New and changed features

- The workflow for importing a single VMware virtual machine is enhanced with (can we be more specific here?)
- ...
- ...

You can now:
- Edit virtual machine templates. Previously, only virtual machines could be edited.
- Add, edit, and view virtual CD-ROM devices attached to a virtual machine.
- ...

Thoughts?

File: cnv/cnv_release_notes/cnv-release-notes.adoc [hunk 3]
```diff
@@ -15,10 +15,20 @@ include::modules/technology-preview.adoc[leveloffset=+2]
...
+
+* Edit Virtual machine templates. Previously, only virtual machines could be edited.
+
+* Health monitoring of Virtual Machine-backed services, equivalent to that for Pod-based services.
```
Comment (ousleyp) [inline cnv/cnv_release_notes/cnv-release-notes.adoc]: s/Virtual Machine/virtual machine

I don't think "equivalent to that for" is the best way to phrase this.

File: cnv/cnv_release_notes/cnv-release-notes.adoc [hunk 4]
```diff
@@ -15,10 +15,20 @@ include::modules/technology-preview.adoc[leveloffset=+2]
...
+
+* Persistent local storage option for virtual machine images.
+
+* The virtual machine creation wizard has been completely redesigned.
+It now uses an in-page style and includes a comprehensive review page.
+
+* Add, edit and view virtual CD-ROM devices attached to a virtual machine.
```
Comment (ousleyp) [inline cnv/cnv_release_notes/cnv-release-notes.adoc]: s/Add, edit and/Add, edit, and

**Review by ousleyp — CHANGES_REQUESTED** [review]
Some comments on the general structure and some phrasing issues. Please let me know when you've got a fresh draft and I'll be happy to review it.

**Comment by ousleyp** [comment]
> This PR adds New Feature bullets edited by PM and Dev.
> 
> @ousleyp Steve G. has said to remove the previous (and only) feature bullet as the dashboard is not new in this release. If you had another reason to include it, please flag in peer review.

I don't think I realized that this was still in the 2.2 RNs. Whoops.

File: cnv/cnv_release_notes/cnv-release-notes.adoc (hunk 1 above)
Comment (bgaydosrh) [reply cnv/cnv_release_notes/cnv-release-notes.adoc]: Done.

File: cnv/cnv_release_notes/cnv-release-notes.adoc (hunk 2 above)
Comment (bgaydosrh) [reply cnv/cnv_release_notes/cnv-release-notes.adoc]: * The virtual machine creation wizard has been completely redesigned.
It now uses an in-page style and includes a comprehensive review page. 
In addition. you can now:
** Edit virtual machine templates as well as virtual machine configurations.
** Import a single VMware virtual machine with less screen navigation.
** Monitor health of Virtual Machine-backed services, as you would for 
Pod-based services.
** Enable persistent local storage for virtual machine images.
** Add, edit and view virtual CD-ROM devices attached to a virtual machine.
** Add and view network attachment definitions with a graphical editor.

If this is OK, I'll commit. Pinging on Slack.

File: cnv/cnv_release_notes/cnv-release-notes.adoc (hunk 3 above)
Comment (bgaydosrh) [reply cnv/cnv_release_notes/cnv-release-notes.adoc]: Rephrased.

File: cnv/cnv_release_notes/cnv-release-notes.adoc (hunk 4 above)
Comment (bgaydosrh) [reply cnv/cnv_release_notes/cnv-release-notes.adoc]: got it.

File: cnv/cnv_release_notes/cnv-release-notes.adoc (hunk 2 above)
Comment (bgaydosrh) [reply cnv/cnv_release_notes/cnv-release-notes.adoc]: Final version:

* Managing virtual machines is simpler and more efficient due to improvements
in design and workflow. You can now:
** Run the virtual machine wizard with less navigation. The wizard now uses
a comprehensive in-page style and includes a review page for confirming
configuration details before submission.
** Import a single VMware virtual machine with less navigation.
** Edit virtual machine templates as well as virtual machine configurations.
** Monitor health of virtual machine-backed services as you would for
Pod-based services.
** Enable persistent local storage for virtual machine images.
** Add, edit, and view virtual CD-ROM devices attached to a virtual machine.
** Add and view network attachment definitions with a graphical editor.

**Review by ousleyp — APPROVED** [review]
Much better. Thanks, Bob! Merging.

**Pull request merged**